parameters/Parameters.jl

```diff
@@ -0,0 +1,55 @@
+# NOTE: This file will soon be automatically generated
+module Parameters
+using CLIMA.ParameterSets
+import CLIMA.ParameterSets.Planet
+
+export EarthParameters
+struct EarthParameters{FT} <: AbstractParameterSet{FT} end
+
+# Properties of dry air
+Planet.molmass_dryair(ps::EarthParameters{FT}) where {FT} = FT(28.97e-3)
```

Comment:
```suggestion
CLIMA.Parameters.Planet.molmass_dryair(ps::EarthParameters{FT}) where {FT} = FT(28.97e-3)
```
and move all these below to `parameters/Planet.jl`